### PR TITLE
Introducing `worlddriven.ini`

### LINKED ISSUE #154: Adding a config file for repository based configuration
Currently the configuration (time per commit, auto merge without reviews, ...) is hardcoded. Different projects have different needs.
A config file which is read from the master branch could allow different projects to have different configurations.

### PR DESCRIPTION
**Pull request by TooAngel**
A `worlddriven.ini` can be placed in the root of the repository to
configure the merge times.
Current options are `baseMergeTimeInHours` and `perCommitTimeInHours`.
The config is fetched from the default branch of the repository and
applied.

Fixes #154

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +27,21 @@ Set Session secret as `SESSION_SECRET` initial a random string.
...
 docker-compose up
 ```
 
+## Configuration
+
+To allow different configurations on different repositories you can place a
+`.worlddriven.ini` (see `.worlddriven.ini-example`) in the root of your
+repository and adapt the values.
+When worlddriven checks new Pull Requests it fetches the `.worlddriven.ini` from
+you default branch (fetching from the Pull Request could be a security issue)
```
Comment (IanFindlay) [inline README.md:42]:
```suggestion
your default branch (fetching from the Pull Request could be a security issue)
```

**Comment by TooAngel** [comment]
Settings this to draft for now, I don't want to introduce merge conflicts with the other PRs

File: README.md (hunk 1 above)
Comment (TooAngel) [reply README.md:42]: Thx, fixed

**Review by IanFindlay — APPROVED** [review]

**Comment by TooAngel** [comment]
This pull request was merged by [worlddriven](https://www.worlddriven.org).

**Pull request merged**